salt/states/file.py

```diff
@@ -6810,7 +6810,7 @@ def copy_(name,
     return ret
 
 
-def rename(name, source, force=False, makedirs=False):
+def rename(name, source, force=False, makedirs=False, **kwargs):
```

Comment:
I do not see `kwargs` being used in this method and the issue associated with this PR is closed. Is this PR obsolete? If not, can you please explain why `kwargs` is needed here?

Reply:
When used with prereq, the same warning from the linked issue is shown, so I applied the same fix. Many other methods add kwargs in the same way, it seems